src/assets/i18n/custom-translations.json

```diff
@@ -127,6 +127,8 @@
   "eventGroup.linkEvent.reportDate": "Report date",
   "eventGroup.linkEvent.filters.sortBy": "Sort by",
   "eventGroup.linkEvent.filters.from": "Event from",
-  "dashboard.newCase.casesWithDefinition": "Cases with reference definotion fulfilled",
-  "dashboard.newCase.rate": "Rate"
+  "dashboard.newCase.casesWithDefinition": "Cases with reference definition fulfilled",
+  "dashboard.newCase.rate": "Rate",
+  "dashboard.newCase.newCases": "New Cases",
+  "dashboard.newEvent.newEvents": "New Events"
```

Comment:
why add new translations when we already have them?
(both New cases and New events exist)

Reply:
Resolved.